assets/ui/taskSelectionScreen.ui

```diff
@@ -1,75 +1,136 @@
+                            "text" : "PICK YOUR MAGIC",
+                            "id"   : "environmentTaskInfoLabel",
 
-                            ]
+                            "layoutInfo": {
+                                "use-content-width": true,
+                                "position-horizontal-center": {
+
+                                },
+                                "height": 30
+                            }
                         },
                         {
                             "type" : "RowLayout",
+                            "id" : "options",
                             "contents" : [
                                 {
-                                    "type" : "UIList",
-                                    "id" : "taskSelectionScreenList"
+                                    "type" : "UIButton",
+                                    "text" : "Tasks",
+                                    "id"   : "tasksTabButton",
+                                    "family" : "title"
+
+
+                                },
+                                {
+                                    "type" : "UIButton",
+                                    "text" : "Buildings",
+                                    "id"   : "buildingsTabButton",
+                                    "family" : "title"
+
                                 }
-                            ]
+                            ],
+                            "layoutInfo": {
+                                "use-content-height": true,
+                                "position-top": {
+                                  "offset": 32
+                                },
+                                "position-horizontal-center": {
+                                  "offset": -5
+                                },
+                                "width":1000
+                              }
                         },
                         {
                             "type" : "RowLayout",
                             "contents" : [
                                 {
-                                    "type" : "UIButton",
-                                    "id" : "cancelButton",
-                                    "text" : "Cancel Selection"
+                                    "type": "ScrollableArea",
+                                    "content": {
+                                        "type" : "UIList",
+                                        "id" : "taskSelectionScreenList",
+                                        "family" : "listAndRef"
+                                    }
+                                },
+                                {
+
+                                    "type" : "UILabel",
+                                    "text" : "REFERENCE PIC",
+                                    "id"   : "environmentTaskInfoLabel",
+                                    "family" : "listAndRef"
```

Comment:
Instead of adding a placeholder `UILabel` you could also add a placeholder `UIImage`. The picture will be available if dropped under `assets/textures` - for this example, I named it `underConstruction.png`):

```json
    "type": "UIImage",
    "image": "MasterOfOreon:underConstruction",
    "id": "environmentTaskInfoLabel",
    "family": "listAndRef"
```

![image](https://user-images.githubusercontent.com/1448874/83976156-926a0980-a8f8-11ea-89a3-2da089f88580.png)

Reply:
Of course you can be more creative with the style of the preview image if you want 😉 🎨